### PR TITLE
[DOCFIX] Add the Chinese version of Contributing-Getting-Started.md

### PR DESCRIPTION
**Pull request by saltylin**

### CODE REVIEW

File: docs/cn/Contributing-Getting-Started.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+---
+layout: global
+title: 开始贡献源码
+nickname: 开始贡献源码
+group: Resources
+---
+
+欢迎来到Alluxio社区，我们十分欢迎你在社区里的参与和贡献。本页面提供了如何参与到Alluxio开源项目社区并成为其中一员的向导。
```
Comment (RongGu) [inline docs/cn/Contributing-Getting-Started.md]: '我们十分欢迎你在社区里的参与和贡献'->'我们热烈欢迎你参与到我们的社区和对社区进行贡献'；‘向导’->'指导'

File: docs/cn/Contributing-Getting-Started.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+## 系统需要
+
+最基本的需要是一台安装了Mac OS X或者Linux的电脑，Alluxio目前还不支持Windows系统。
```
Comment (RongGu) [inline docs/cn/Contributing-Getting-Started.md]: ‘Alluxio目前还不支持Windows系统’ ->'目前版本的Alluxio尚未对Windows系统提供支持'

**Comment by RongGu** [comment]
Thanks for the contributation. I have some minor comments. Please handle that, thanks! @saltylin

File: docs/cn/Contributing-Getting-Started.md (hunk 1 above)
Comment (saltylin) [inline docs/cn/Contributing-Getting-Started.md]: Updated

File: docs/cn/Contributing-Getting-Started.md (hunk 2 above)
Comment (saltylin) [inline docs/cn/Contributing-Getting-Started.md]: Updated

**Comment by saltylin** [comment]
Thanks for your review, and I have addressed your comments!@RongGu

**Comment by RongGu** [comment]
LGTM.

**Comment by saltylin** [comment]
Jenkins, test this please

File: docs/cn/Contributing-Getting-Started.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
+---
+layout: global
+title: 开始贡献源码
```
Comment (ifcharming) [inline docs/cn/Contributing-Getting-Started.md]: nit: 社区贡献入门指南？

File: docs/cn/Contributing-Getting-Started.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+你需要一个GitHub账户来贡献源码，如果还没有，请先[注册一个GitHub账户](https://github.com/)。
+
+另外你还需要知道[你的GitHub账户绑定了哪个邮箱](https://help.github.com/articles/adding-an-email-address-to-your-github-account/)。
+
+### Jira账户
+
+Alluxio开源项目使用[JIRA](https://alluxio.atlassian.net/projects/ALLUXIO/issues)来跟踪特性、bug以及问题。请[创建一个Alluxio JIRA账户](https://alluxio.atlassian.net/admin/users/sign-up)，从而能够新建、跟踪以及修复ticket。
+
+## Fork Alluxio仓库
```
Comment (ifcharming) [inline docs/cn/Contributing-Getting-Started.md]: s/仓库/代码库？

**Comment by ifcharming** [comment]
@saltylin  thank you for the translation! I left a few minor comments.

**Comment by apc999** [comment]
LGTM.
@ifcharming I will address your comments in a followup PR since they are minor.
Merge this first

**Pull request merged**